### DISCUSSION TITLE
wire:poll - Max number of requests

### DISCUSSION THREAD

**Discussion by aran112000** [livewire/livewire#5532] [3. Feature ideas]
In my experience, it's a common scenario when using `wire:poll` to need to be following an update process, and polling indefinitely for changes can be very wasteful. 

I'd like to propose a new optional directive of `max:x` taking an integer as the maximum number of requests to perform before the polling halts that'd look something like this:

```html
<div wire:poll.max:60>...</div>
```

**Comment by ChrisSantiago82** [comment]
Couldn't you achieves this with a simple if statement and a variable with a counter?

Or when it got updated you could even remove the poll completely
